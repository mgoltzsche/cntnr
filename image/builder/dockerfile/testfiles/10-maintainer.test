FROM alpine:3.7
MAINTAINER Max Goltzsche <max.goltzsche@gmail.com>

# ASSERT CFG /author=Max Goltzsche <max.goltzsche@gmail.com>