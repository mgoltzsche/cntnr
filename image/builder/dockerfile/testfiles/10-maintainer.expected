FROM alpine:3.7
AUTHOR "Max Goltzsche <max.goltzsche@gmail.com>"